libsupport/include/katana/Result.h

```diff
@@ -639,6 +639,16 @@ CheckedExpressionToValue(arrow::Status&&) {
 /// function calling KATANA_CHECKED_CONTEXT will return the error with
 /// additional error formatting. Otherwise, KATANA_CHECKED_CONTEXT will return
 /// the value of the Result object to the caller.
+///
+/// Because KATANA_CHECKED_CONTEXT is a macro, it will interpret all commas that
+/// are not enclosed in a pair of parentheses as delimiters for its own
+/// arguments. This is problematic for functions with multiple template
+/// parameters. The upshot of this is that
+/// ``KATANA_CHECKED_CONTEXT(MakeSword<Handle, Blade>(), "third argument")``
```

Comment:
There are multiple levels of markup interpretation Doxygen -> Sphinx (RST).

If you want to show a code block in Doxygen, you can follow the example in this file and insert a new line and then prefix with 4 spaces.

Reply:
I changed to using Doxygen style inline code formatting. If I understand correctly, Doxygen will parse that to a Doxygen representation of inline code and rst will respect that. So it should render as code in both sets of docs?

Reply:
I personally prefer a code block (new line, prefix with 4 spaces) for this as it is easier to read in raw form and in a rendered document.

And for your example, it would probably be confusing if the example were wrapped between two lines.